.cspell.json

```diff
@@ -1525,6 +1525,7 @@
     "tfar",
     "TFLOPS",
     "theora",
+    "Thinkpad",
```

Comment:
The official notation is `ThinkPad`, so it's good to add the lowercase word, considering the usage when written in URLs (as your reference link).

```suggestion
    "thinkpad",
```

Reply:
@Yukio247 What do you think?

Reply:
@kenji-miyake 
Thank you. It would be fine with me!